app/scripts/modules/amazon/src/serverGroup/configure/wizard/advancedSettings/advancedSettings.component.html

```diff
@@ -104,6 +104,16 @@
     </div>
   </div>
   <div class="form-group">
+    <div class="col-md-5 sm-label-right">
+      <b>Spot Instances Price (optional)</b>
+      <help-field key="aws.serverGroup.spotPrice"></help-field>
+    </div>
+    <div class="col-md-2"><input type="text"
+                                 class="form-control input-sm no-spel"
```

Comment:
Is there a reason to not support SPEL for spotPrice?

Reply:
What is SPEL? The Spinnaker Expression Language? 

If so, then I didn't add it in because the thought never occurred to me. Good idea though! I'll take a look at it.

Reply:
I removed the `no-spel` class from the input